runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/options/DataflowWorkerLoggingOptions.java

```diff
@@ -38,7 +38,47 @@
    * The set of log levels that can be used on the Dataflow worker.
    */
   public enum Level {
-    DEBUG, ERROR, INFO, TRACE, WARN
+    /**
+     * Special level used to turn off logging.
+     *
+     * @see java.util.logging.Level#OFF
```

Comment:
I disagree that JUL is an internal-only implementation detail since the Level filtering logic is tightly coupled to JUL's implementation. However, I agree that we shouldn't unnecessarily add this doc dependency between SDK and service implementation. I'll remove the JUL ref.